xml/System/DateOnly.xml

```diff
@@ -239,7 +239,7 @@
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets the day component of the date represented by this instance.</summary>
         <value>To be added.</value>
```

Comment:
@tarekgh there are several `<value>` elements missing. Can you please review my suggestions?:

```suggestion
        <value>A number representing the day component of the date represented by this instance.</value>
```

Reply:
Your suggestion looks good but I am wondering should we do that? I mean the summary already has the needed information and the value description is just a redundant.

Reply:
Maybe I'm not providing the best `<value>` description for this Property. The `<value>` element is supposed to describe the data type and the potential values the property can return. It's usually more noticeable for boolean properties.

One reason why I need these added is so the undoc APIs report does not show these as undocumented.

I find it strange that the autogenerated triple slash comments you can add with Visual Studio, never include the `<value>` item for a property, yet MS Docs sees it as valuable.

I'll talk about this with the Docs team.